guiEditor/src/diagram/workbench.tsx

```diff
@@ -28,6 +28,9 @@ import { NodeMaterial } from "babylonjs/Materials/Node/nodeMaterial";
 import { TextureBlock } from "babylonjs/Materials/Node/Blocks/Dual/textureBlock";
 import { Observer } from "babylonjs/Misc/observable";
 import { GUIEditorNodeMaterial } from "./GUIEditorNodeMaterial";
+
+const DOUBLE_CLICK = Scene.DoubleClickDelay;

```

Comment:
Why a custom one here, maybe just directly always reuse the scene one so custo would apply accross the board ?